mmv1/products/redis/api.yaml

```diff
@@ -400,3 +400,54 @@ objects:
           an existing instance. For DIRECT_PEERING mode value must be a CIDR range of size /28, or
           "auto". For PRIVATE_SERVICE_ACCESS mode value must be the name of an allocated address 
           range associated with the private service access connection, or "auto".
+        input: true
```

Comment:
Is this affecting the wrong field? I'm not sure why `secondaryIpRange` needs to change.